File: Pascal/Lab1/Variant4.pas
Program lab1var4;
Uses Crt;
Var x,y,Formula: real;
Begin
  Write('Введите первое число: '); Readln(x);
  Write('Введите второе число: '); Readln(y);
  ClrScr;
  Formula:=1+x+sqr(x)/sin(x)+power(x,3)/cos(x)+power(x,4)/(sin(x)/cos(x));
  Writeln('Вариант 4.  ',Formula);
End.

File: Python/Lab1/Variant 4.py
from math import *
x = float(input("Введите значение x: "))
y = float(input("Введите значение y: "))
s = 1+x+(x**2)/sin(x)+(x**3)/cos(x)+(x**4)/tan(x)
print("\nЗначение s =", s)
